misc/python/materialize/ui.py

```diff
@@ -27,7 +28,7 @@ class Verbosity:
     quiet: bool = False
 
     @classmethod
-    def init_from_env(cls, explicit: Optional[bool]) -> None:
+    def init_from_env(cls, explicit: bool | None) -> None:
```

Comment:
This line is giving me an error when running `bin/environmentd`

```
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.9/3.9.13_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/homebrew/Cellar/python@3.9/3.9.13_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/dan/mz/materialize/misc/python/materialize/cli/run.py", line 24, in <module>
    from materialize import MZ_ROOT, rustc_flags, spawn, ui
  File "/Users/dan/mz/materialize/misc/python/materialize/spawn.py", line 24, in <module>
    from materialize import ui
  File "/Users/dan/mz/materialize/misc/python/materialize/ui.py", line 25, in <module>
    class Verbosity:
  File "/Users/dan/mz/materialize/misc/python/materialize/ui.py", line 31, in Verbosity
    def init_from_env(cls, explicit: bool | None) -> None:
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```

Reply:
@danhhz you may need to ensure you are using Python 3.11 😢

Reply:
Looks like I do have 3.11 installed via homebrew and `python3` is even pointed at it (`python` seems to point at nothing for me).

```
$ python3 --version
Python 3.11.3
$ python --version
-bash: python: command not found
```

I really have no idea how bin/environmentd picks the python version it uses. The only thing I could find is that pyactivate (used by bin/run used by bin/environmentd) seems requiring 3.9.0+. Maybe that has something to do with it?

https://github.com/MaterializeInc/materialize/blob/main/bin/pyactivate#L31-L32

Reply:
Hrnmmmm, lemme look at this in a minute

Reply:
It uses your system Python version at the time you first run `pyactivate`. You almost certainly have a virtualenv from way back when, when your system Python was Python 3.9. You can fix this like so:

```shell
rm -rf misc/python/env
# <run any command that uses pyactivate>
bin/environment
``

Reply:
That worked, thanks!